### PR TITLE
Update UI filtering

### PR DESCRIPTION
**Pull request by AlexGalchenko**

### CODE REVIEW

File: templates/default/views/DashboardView.html [hunk 1]
```diff
@@ -26,30 +26,42 @@
...
+                 type="search" 
+                 class="form-control" 
+                 ng-model="query" 
+                 uib-typeahead="name as tc.name for tc in testCases | filter:$viewValue | limitTo:8" 
+                 typeahead-on-select="setQueryFilterValue($item.name)" 
+                 placeholder="Test Case Filter" 
+              />  
+            </div>
+            <div class="col-sm-2 checkbox">
+               <label for="excludeFilter"><input type="checkbox" id="excludeFilter" ng-model="excludeFilter" ng-checked="excludeFilter"><code> Exclude</code></label>
```
Comment (eger-geger) [inline templates/default/views/DashboardView.html]: According to [angular](https://docs.angularjs.org/api/ng/directive/ngChecked) _ng-checked_ should not be used together with _ng-model_

File: templates/default/views/DashboardView.html [hunk 2]
```diff
@@ -26,30 +26,42 @@
...
     <ul class="list-group">
         <a 
             class="list-group-item" 
             ng-class="{
               'list-group-item-success': tc.result == 'Passed',
               'list-group-item-danger': tc.result == 'Failed',
               'list-group-item-warning': tc.result == 'Skipped',
-              'list-group-item-info': tc.result == 'Inconclusive'
+              'list-group-item-info': tc.result == 'Inconclusive',
+              'selected': tc.id === currentTestCase.id
```
Comment (eger-geger) [inline templates/default/views/DashboardView.html]: You could probably use one of the bootstrap classes for it.

File: templates/default/views/DashboardView.html [hunk 3]
```diff
@@ -26,30 +26,42 @@
...
             }" 
             ng-click="openTestCase(tc)" 
-            ng-repeat="tc in testCases | filter: query | filter: testCaseFilter"
+            ng-repeat="tc in testCases | filter : testCaseFilter | filter: textFilter"
```
Comment (eger-geger) [inline templates/default/views/DashboardView.html]: Please rename to _testCaseResultFilter_ and _testCaseTextFilter_

**Review by eger-geger — CHANGES_REQUESTED** [review]
Please take a look on comments before merge

File: templates/default/views/DashboardView.html (hunk 1 above)
Comment (AlexGalchenko) [inline templates/default/views/DashboardView.html]: changed to ng-attr-checked

File: templates/default/js/controllers/DashboardController.js [hunk 4]
```diff
@@ -1,9 +1,14 @@
 var RESULT_TYPE_ANY = "Any";
 
 function DashboardController($scope, $location, $routeParams, StateStorage, reportAdapter) {
-    $scope.testCaseFilter = function(testCase) {
+	
+	$scope.testCaseTextFilter = function(testCase) {
```
Comment (eger-geger) [inline templates/default/js/controllers/DashboardController.js]: It does not filter by test case text but based on test status

File: templates/default/views/DashboardView.html (hunk 1 above)
Comment (AlexGalchenko) [inline templates/default/views/DashboardView.html]: removed

**Review by eger-geger — APPROVED** [review]

**Pull request merged**